webapp/packages/core-server-localization/src/ServerLocalizationService.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { Dependency, injectable } from '@cloudbeaver/core-di';
+import { LocalizationService } from '@cloudbeaver/core-localization';
+import { ServerConfigResource } from '@cloudbeaver/core-root';
+
+@injectable()
+export class ServerLocalizationService extends Dependency {
+  constructor(private readonly serverConfigResource: ServerConfigResource, private readonly localizationService: LocalizationService) {
+    super();
+
+    this.serverConfigResource.onDataUpdate.addHandler(this.updateSupportedLocalization.bind(this));
+  }
+
+  private updateSupportedLocalization() {
+    const supportedLanguages = this.serverConfigResource.data?.supportedLanguages;
+
+    if (supportedLanguages) {
+      this.localizationService.setSupportedLanguages(
+        supportedLanguages.map(lang => ({
+          isoCode: lang.isoCode,
+          displayName: lang.displayName ?? lang.nativeName ?? lang.isoCode,
+        })),
+      );
+    }
+  }
```

Comment:
we can find it by trying to switch to another language (we will see only english option)
actually we don't want to display any notifications to users because we will try to fall back to english and give users access to application
they will notice that language is not as should be in case they use other langoages than english